test/Function/StackArgsWithFormals.js

```diff
@@ -0,0 +1,141 @@
+//Test1 : Tests Optimization with bailout.
+var shouldBailout = false;

```

Comment:
At first look, it doesn't seem to me that i can run each function twice, for jit to kick in ?

---
In reply to: [67765923](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67765923) [](ancestors = 67765923)